util/MigratorCLI/MigratorCLI.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
ℹ️ Also, we didn't know this at the time of course but the migrator project is really the "MSSQL migrator". Could be nice to call this the "MssqlMigratorUtility" or something. The term "CLI" is a stretch to me and this is very much a console / utility.

Reply:
Renamed :)